src/agora/stats.cc

```diff
@@ -6,19 +6,23 @@
 
 #include <typeinfo>
 
-Stats::Stats(Config* cfg)
+Stats::Stats(const Config* const cfg)
     : config_(cfg),
       task_thread_num_(cfg->WorkerThreadNum()),
       fft_thread_num_(cfg->FftThreadNum()),
       zf_thread_num_(cfg->ZfThreadNum()),
       demul_thread_num_(cfg->DemulThreadNum()),
       decode_thread_num_(cfg->DecodeThreadNum()),
       freq_ghz_(cfg->FreqGhz()),
-      creation_tsc_(Rdtsc()) {
+      creation_tsc_(GetTime::Rdtsc()),
+      doer_us_(),
+      doer_breakdown_us_() {
```

Comment:
Is it necessary to have line 18 and 19 here?

Reply:
Either I should have all the class members in the list, or remove all the default constructed members.  Effective C++ recommends all members be explicitly initialized to avoid errors and reliance on c++ language standards.   For this PR, I will remove these two definitions and if we wish to follow this guidance we can make all the classes complaint in a future update.

Reply:
Removed the to default ones for this pr.